src/main/java/uk/gov/hmcts/darts/annotation/service/impl/AnnotationDownloadServiceImpl.java

```diff
@@ -45,12 +45,11 @@ public AnnotationResponseDto downloadAnnotationDoc(Integer annotationId, Integer
 
         final ExternalObjectDirectoryEntity latestExternalObjectDirectoryEntity = eodDir.get(0);
 
-        final InputStreamResource blobStream = annotationDataManagement.download(latestExternalObjectDirectoryEntity);
+        final InputStreamResource blobStream = annotationDataManagement.download(eodDir);
 
         return AnnotationResponseDto.builder()
             .resource(blobStream)
             .fileName(latestExternalObjectDirectoryEntity.getAnnotationDocumentEntity().getFileName())
```

Comment:
I know we should have the annotation_document from one of the external_object_directory links, but it feels better to me to use a `findById` here with the `annotationDocumentId` param.
So then we wouldn't need `latestExternalObjectDirectoryEntity` on line 46?

I'll let you decide if you agree on doing that here :)

Reply:
yeah - the EODs are returned in descending order so the first one is the latest. the database supports multiple EOD with same annotation id. I've decided to leave as is :-)